views/question/show.ejs

```diff
@@ -37,23 +26,24 @@
 
 <br>
 <div class="container">
-    <form>
-        <h1 class="prompt"> <%= record.step %>.  <%= record.prompt %></h1>
-
-        <ul class="answers">
-            <div class="form-check">
-                <input class="form-check-input" type="radio" name="exampleRadios" id="exampleRadios1" value="option1" checked>
-                <label class="form-check-label answer" for="exampleRadios1">
-                    <%= record.answer_type%>
-                </label>
-            </div>
-        </ul>
-
-
-        <br>
-        <br>
-        <button class="btn btn-primary float-right" type="submit">Contiue</button>
-    </form>
+    <!-- current question -->
+        <h1 class="prompt"> <%= question.step %>.  <%= question.prompt %></h1>
+        <!-- load available answers -->
+        <% for (let answer of answers) { %>
+            <% if (answer.QuestionId == question.id) { %>
+        <form action="/questions/<%= answer.NextQuestionId %>">
+            <ul class="answers">
+                    <div class="form-check">
+                        <input class="form-check-input" type="radio" >
+                        <label class="form-check-label answer" for="<%= answer.NextQuestionId %> ">
+                            <%= answer.value %>
+                        </label>
+                    </div>
+                    <% } %>
+                <% } %>
+            </ul><br><br>
+            <button class="btn btn-primary float-right" type="submit">Contiue</button>
+        </form>
```

Comment:
Can you fix the indentation so that we can see the nesting of the tags? Right now this is generating some invalid HTML. 
<img width="722" alt="Screen Shot 2020-03-25 at 5 34 25 PM" src="https://user-images.githubusercontent.com/140766/77598413-f9ae1b00-6ebe-11ea-9ff8-511c21b4f4bc.png">

Reply:
Fixed this issue.It now generates valid html